src/forms/AdminUserForm.tsx

```diff
@@ -33,7 +33,7 @@ const AdminUserForm = createForm({
     address: v.user?.address ?? '',
     non_giver: v.user?.non_giver ?? false,
     fixed_non_receiver: !!v.user?.fixed_non_receiver ?? false,
-    non_receiver: !!v.user?.fixed_non_receiver || !!v.user?.non_receiver,
+    non_receiver: !!v.user?.fixed_non_receiver || !!v.user?.non_receiver || !v.circle.default_opt_in,
```

Comment:
Won't this change all existing users to `non_receiver=true` if the circle has `default_opt_in=false`?

Reply:
I am using this to make the best guess about the initial value for the corresponding `Checkbox`. Consequently, if the circle's default value is to opt out it won't be checked, admin can decide then and vice versa. 

I can't get how it's going affect user(s) outside the Add/Edit User Modal.